src/custom-compiler.ts

```diff
@@ -0,0 +1,15 @@
+import * as __typeof__ts from 'typescript';

```

Comment:
this seems like a really strange name, also given the typeof __typeof_ts statement below - can we make this clearer somehow?

Reply:
I like your `ts_types` suggestion.